parteh/PRTAllometricCarbonMod.F90

```diff
@@ -51,6 +51,9 @@ module PRTAllometricCarbonMod
 
   use PRTParametersMod    , only : prt_params
 
+  use EDTypesMod          , only : leaves_on
```

Comment:
my original intention with PARTEH was to have it act as a completely independent module, with no references to FATES. This would allow it to be used by other driving models.  I'm not sure that is worth it anymore... I think there are also workarounds... We could simply have a FatesPRTGlobals module that just re-directs things like the leaves_on constant to EDTypesMod.  And if another module wanted to use PARTEH, it could do something similar (ie like ED2PRTGlobals would redirect "leaves_on" to whatever integer they use to define an "on" state).

Reply:
Since we are already doing this for the CNP version of the code, and a precident has been set, I think it is out of scope to fix this issue in this PR.  So lets just proceed using the EDTypesMod global leaves_on/leaves_off for now